backend/src/app/models.py

```diff
@@ -22,37 +24,48 @@ class BaseDAO:
     )
 
     @classmethod
-    def new(cls, session: sa.orm.Session, api_model: Union[BaseModel, dict]):
+    async def new(cls, session: AsyncSession, api_model: Union[BaseModel, dict]):
         if isinstance(api_model, BaseModel):
             api_model = api_model.dict()
 
         model = cls(**api_model)
         session.add(model)
+        await session.commit()
```

Comment:
Why committing here in the innards?

Reply:
Good question, that's how Gate's `BaseDAO` was written.  Does it need to be committed here in order to be flushed/refreshed before being returned in `create`?

Reply:
I take it back, that's now how `BaseDAO.new` works, removing the commit call.